### DISCUSSION TITLE
How to add #tags to file name?

### DISCUSSION THREAD

**Discussion by WarmSleet** [windingwind/zotero-actions-tags#153] [Ideas]
I use 2-3 #tags to summarise the main points, like  #policy #design etc.
I am wondering if it is possible to [add #policy #design in file name with the help of zotfile] or [add #tags to specific field, like add #tags to the front of the title]
this will help me a lot if I want to export all the files and organize files using #tags.

**Comment by windingwind** [comment]
I don't think it's a good idea to use fields in that way. Tags should just be tags.

Close here.
